### DISCUSSION TITLE
Why DCDAO's twitter is posting more contents unrelated to Darwinia and other project.

### DISCUSSION THREAD

**Discussion by hackfisher** [ringdao-community-guild/discussion#64] [General]
It is clearly detail that 

> The Darwinia Community DAO (DCDAO) is a community self-organized workgroup (WG) that aims to contribute to the growth and development of the Darwinia community.

https://snapshot.org/#/darwiniacommunitydao.eth/proposal/0x257aff1540719475411091706f7b200507fe9b9fe7397a8a4347fbc9d7234cb6

For example: https://twitter.com/official_dcdao/status/1744909895751840149?s=46&t=VOVXNP3t4D6F0A0_wmAsmA

Then why it is posting unrelated infos to Darwinia, @Cindy884 can you give some explanation about your operation of this twitter handle and your strategy. Is it delegated to anyone else?

**Comment by wuminzhe** [comment]
account stolen?

**Comment by SasoLithops** [comment]
**Jesse posted fwd this post he got:**


Hi Team! Happy happy new years! Hope it was a memorable one! 😉 We have just shared a campaign highlighting the brand new features of IdentityHub V5+6. If you dont mind, would be great if you can re-tweet the post for some attention! 😉 Many thanks in advance! 

https://twitter.com/litentry/status/1744329044555100566

Best wishes for 2024 ⭐️

**And asked Cindy:**

@CindyTan79 Think we have HRMP channels open with Litentry.  Can we help out with a retweet?


**I think it is ok what they did. It is ok if we help our partners☺️ Would be great if some shared ours**

**Reply by hackfisher** [reply]
I see, thanks for the explanation, I asked that because the tweet was unrelated to the partner scope related to darwinia, the content confused me.

**Reply by 2xJDubs** [reply]
We also held a Space and giveaway with Litentry last year.  So my assumption was that we continue to support a partner we had worked with.  

https://x.com/litentry/status/1733103488131858876?s=20

They were a little disheartened that the retweet did not come from Darwinia.  I received this message in the group chat with Litentry:
"Hello Team Darwinia,

There may be a situation that requires some communication with you. We've collaborated on an event before, and if I remember correctly, you have an official Twitter account with a larger following. Our initial expectation for the retweet was that, if you could help us promote our campaign, it would be done through the larger account and not a smaller one.

Currently, we have specific requirements for retweets on our Twitter. If the content being retweeted is not related to our product/brand collaboration, we have certain expectations regarding the follower count of the collaborating project. This is based on considerations for our user experience.

Thank you for your understanding🙏"
